Airports/EGOV/Positions.txt

```diff
@@ -1,4 +1,4 @@
-EGOV_APP:Valley Approach:125.220:OVA:R:EGOV:APP:-:-:3720:3765:N053.14.52.000:W004.32.03.000::
+EGOV_APP:Valley Approach:125.220:OVR:R:EGOV:APP:-:-:3720:3765:N053.14.52.000:W004.32.03.000::
```

Comment:
Does it provide radar? I see in the AIP there's a director frequency?

Reply:
Military unit gives full radar. Remains as approach. No VHF freq for director available and not worth making one up.